src/metabase/db/migrations.clj

```diff
@@ -20,24 +19,23 @@
 
 ;;; # Migration Helpers
 
+(defentity DataMigrations :data_migrations)
+
 (defn- migration-ran? [migration-name]
-  (-> (k/select :data_migrations
-                (k/aggregate (count :*) :count)
-                (k/where {:id (name migration-name)}))
-      first :count (> 0)))
+  (db/exists? DataMigrations :id (name migration-name)))
```

Comment:
hurray for getting rid of a big ass query that was spelled out for no real reason.